themes/default/components/src/components/convert/convert.e2e.ts

```diff
@@ -1,11 +1,11 @@
-import { newE2EPage } from '@stencil/core/testing';
+import { newE2EPage } from "@stencil/core/testing";
 
-describe('pulumi-convert', () => {
-  it('renders', async () => {
-    const page = await newE2EPage();
-    await page.setContent('<pulumi-convert></pulumi-convert>');
+describe("pulumi-convert", () => {
+    it("renders", async () => {
+        const page = await newE2EPage();
+        await page.setContent(`<pulumi-convert from="tf" endpoint="https://someserver/somepath"></pulumi-convert>`);
```

Comment:
To quiet the `pulumi-convert` test, which was complaining about these attributes being missing.